src/style.css

```diff
@@ -1,3 +1,86 @@
+
 body {
   margin: 0;
+  padding: 0;
+  background-color: #ebebe4;
+  font-family: Roboto, Inter, Helvetica, sans-serif;
+  box-sizing: border-box;
+  --color-selected: #00acdc;
+}
+
+.header {
+  display: flex;
+  margin: auto;
+  justify-content: space-between;
+  align-items: center;
+  min-width: 1200px;
```

Comment:
Use a combination of width: 100% to force the element to take all parent widths but also add max-width: 1200px to limit it. Look at the visual bug for your current approach.
![image](https://github.com/mate-academy/layout_moyo-header/assets/45945732/a8673a03-78ec-4dc3-ba4d-7ee00eae128b)